src/RadixDlt.CoreApiSdk/Model/LedgerTransaction.cs

```diff
@@ -78,4 +78,19 @@ public byte[] GetPayloadBytes()
             _ => throw new UnreachableException($"Didn't expect {this.GetType().Name} type"),
         };
     }
+
+    // TODO:
+    // This is rcnet quick fix for returning transaction hex that's not boxed in user/validator/system enum by node.
+    // Boxed payload is not supported by the toolkit and client's can't actually do anything about it.
+    // Needs revisiting.
+    public byte[] GetUnboxedPayloadBytes()
```

Comment:
Might suggest "Unwrapped" rather than "Unboxed" because boxing has a special meaning in C#/programming, but otherwise good!